src/Example/UserRegistrationBundle/Entity/User.php

```diff
@@ -0,0 +1,289 @@
+<?php
+/*
+ * Copyright (c) 2012, 2014 KUBO Atsuhiro <kubo@iteman.jp>,
+ *               2014 YAMANE Nana <shigematsu.nana@gmail.com>,
+ * All rights reserved.
+ *
+ * This file is part of PHPMentors_Training_Example_Symfony.
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the BSD 2-Clause License which accompanies this
+ * distribution, and is available at http://opensource.org/licenses/BSD-2-Clause
+ */
+
+namespace Example\UserRegistrationBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+use PHPMentors\DomainKata\Entity\EntityInterface;
+
+/**
+ * @ORM\Entity(repositoryClass="Example\UserRegistrationBundle\Repository\UserRepository")
+ * @ORM\Table(name="user",
+ *      uniqueConstraints={
+ *          @ORM\UniqueConstraint(name="user_email_idx", columns={"email"}),
+ *          @ORM\UniqueConstraint(name="user_activation_key_idx", columns={"activation_key"})
+ *      })
+ * @ORM\HasLifecycleCallbacks
+ * @UniqueEntity(fields="email", message="このメールアドレスはすでに使用されています", groups={"registration"})
+ */
+class User implements EntityInterface
+{
+    const SALT = 'cbab20bf0631558e0b723e5f48c337237a4d862d';
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
```

Comment:
__[development][phpmd]__
* Avoid variables with short names like $id. Configured minimum length is 3.